Documentation/ceph-csi-drivers.md

```diff
@@ -92,3 +92,49 @@ kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replicatio
 2. Enable the volume replication controller:
    - For Helm deployments see the [csi.volumeReplication.enabled setting](helm-operator.md#configuration).
    - For non-Helm deployments set `CSI_ENABLE_VOLUME_REPLICATION: "true"` in operator.yaml
+
+## generic ephemeral volume support
+
+The generic ephemeral volume feature adds support for specifying PVCs in the
+`volumes` field to indicate a user would like to create a Volume as part of the pod spec.
+This feature requires the GenericEphemeralVolume feature gate to be enabled.
+Because this is a beta feature, it is enabled by default in kubernetes versions >=1.21.
+
+For example:
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: csi-rbd-demo-ephemeral-pod
+spec:
+  containers:
+    - name: web-server
+      image: docker.io/library/nginx:latest
+      volumeMounts:
+        - mountPath: "/myspace"
+          name: mypvc
+  volumes:
+    - name: mypvc
+      ephemeral:
+        volumeClaimTemplate:
+          spec:
+            accessModes: ["ReadWriteOnce"]
+            storageClassName: "rook-ceph-block"
+            resources:
+              requests:
+                storage: 1Gi
+```
+
+A volume claim template is defined inside the pod spec which refers to a volume
+provisioned and used by the pod with its lifecycle. The volumes are provisioned
+when pod get spawned and destroyed at time of pod delete.
+
```

Comment:
For links, how about this? The links are generated automatically by the website based on the release docs version.
```
See the example manifests for an [RBD ephemeral volume](https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/rbd/pod-ephemeral.yaml)
and a [CephFS ephemeral volume](https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/cephfs/pod-ephemeral.yaml).
```

For these links to work, you'll also need to include this snippet at the top of this doc, as seen in other rook docs:
```
{% include_relative branch.liquid %}
```

Reply:
better I removed this line completely